osu.Game.Tests/Visual/Online/TestSceneNewsOverlay.cs

```diff
@@ -1,23 +1,74 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
 using osu.Game.Overlays;
+using osu.Game.Overlays.News;
 
 namespace osu.Game.Tests.Visual.Online
 {
     public class TestSceneNewsOverlay : OsuTestScene
     {
-        private NewsOverlay news;
+        private TestNewsOverlay news;
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            Add(news = new NewsOverlay());
+            Add(news = new TestNewsOverlay());
             AddStep(@"Show", news.Show);
             AddStep(@"Hide", news.Hide);
 
             AddStep(@"Show front page", () => news.ShowFrontPage());
             AddStep(@"Custom article", () => news.Current.Value = "Test Article 101");
+
+            AddStep(@"News article covers", () => news.ShowContent(new TestNewsContent()));
+        }
+
+        private class TestNewsOverlay : NewsOverlay
+        {
+            public TestNewsOverlay()
+            {
+            }
+
+            public void ShowContent(NewsContent content) => LoadChildContent(content);
+        }
+
+        private class TestNewsContent : NewsContent
+        {
+            public TestNewsContent()
+            {
+                Spacing = new osuTK.Vector2(0, 10);
```

Comment:
This is going to be quite a nit, but why not just import the namespace? I don't see any potential pollutions.